lox_services/persistence/database/query_handlers.py

```diff
@@ -115,25 +139,25 @@ def update(query: str, print_query: bool = True) -> int:
         raise Exception("Error processing update: ", query)
 
 
-def delete(query: str, print_query: bool = True) -> DataFrame:
+def delete(query: str, print_query: bool = True, *, parameters:  Optional[Sequence[Tuple[str, BQParameterType, Any]]] = None) -> DataFrame:
     """Checks if the query begings with a DELETE statement. If so the query is being executed.
     
         ## Arguments
         
         - `query`: String representation of the query to be executed.
-        
         - `print_query`: Tells whether or not to print the query before executing it.
+        - `parameters`: list of parameter #used to avoid sql injection
         
         ## Example
         
-            >>> select("DELETE InvoicesData.Refunds WHERE tracking_number = '1467BDYV'")
-
+            >>> delete("DELETE FROM InvoicesData.Refunds WHERE tracking_number = '1467BDYV'")
+            >>> delete("DELETE FROM InvoicesData.Refunds WHERE tracking_number=@tracking_number", ("tracking_number", "STRING", "1467BDYV"))
```

Comment:
If your type hints are correct, then this code would not work. 

 `parameters = [("tracking_number", BQParameterType.STRING, "1467BDYV")])`